SWLOR.Game.Server/Conversation/EditPrimaryResidence.cs

```diff
@@ -262,7 +285,17 @@ private void DoSetAsResidence()
             ClearNavigationStack();
             ChangePage("MainPage", false);
         }
-
+        private void DoSetRespawnPoint()
```

Comment:
Does this actually work inside of the building? I don't think respawn locations can be saved if the player is inside an instance.

Also, does this work for both buildings AND apartments or just buildings? I think there are separate convos for each maybe?

Reply:
I've tested with an apartment and it set my respawn position. I may have to do some more testing but I would assume it would work the same way as teleporting you to your instance

Reply:
Just tested and it does work in the building